wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```
@@ -1235,6 +1268,49 @@ private static function getLogisticsEmailHtml($contactName, $collectionCampId, $
+        This feedback form should be filled out after the camp/drive ends, once you have an overview of the event's outcomes.</li>
+    </ol>
+    <p>We appreciate your cooperation.</p>
+    <p>Warm Regards,<br>Urban Relations Team</p>";
+
+    return $html;
+  }
+
+  /**
+   *
+   */
+  private static function getSelfOutcomeLogisticsEmailHtml($contactName, $collectionCampId, $campAttendedById, $collectionCampGoonjOffice, $campCode, $campAddress) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$collectionCampGoonjOffice'. (undefined)

(UnusedFormalParameter)